packages/arb-provider-go/test/eventListeners.go

```diff
@@ -0,0 +1,68 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"log"
+)
+
+func StartEventListeners(fibonacci *Fibonacci) chan interface{} {
+	ch := make(chan interface{}, 64)
+
+	startGenerateFibListener(fibonacci, ch)
+	//startSubmitYourDeckHashListener(poker, ch)
+	//startEncryptAndProveCommandListener(poker, ch)
+	//startDealFaceDownCardListener(poker, ch)
+	//startDealFaceUpCardListener(poker, ch)
+	//startCallForBetActionListener(poker, ch)
+	//startPresentBestHandListener(poker, ch)
+	//startDeclareWinningsListener(poker, ch)
+	//startReportZKPListener(poker, ch)
```

Comment:
This comments should be removed and startGenerateFibListener should probably just be inlined since there's only one function. It might be even easier to just kill this file entirely and move it into the test since there's not much code here.